developer-docs-site/static/messageboard_demo/client/README.md

```diff
@@ -0,0 +1,187 @@
+# MessageBoard
+
+### Setup and Run the Demo
+
+Download Aptos Core
+
+```bash
+git clone https://github.com/aptos-labs/aptos-core.git
+cd aptos-core
+./scripts/dev_setup.sh
+source ~/.cargo/env
+```
+
+Install Apto Cli [[doc](https://github.com/aptos-labs/aptos-core/tree/main/crates/aptos)]
+
+```bash
+cargo install --git https://github.com/aptos-labs/aptos-core.git aptos
+```
+
+Run demo using the commands below 
```

Comment:
Run demo using the below commands:

Reply:
colons are needed here because we are pointing to something.

Reply:
added